docs/docs/vm/aws.md

```diff
@@ -25,6 +25,12 @@ $ trivy vm --security-checks vuln ami:ami-0123456789abcdefg
 !!! tip
     The scanning could be faster if you enable only vulnerability scanning (`--security-checks vuln`) because Trivy tries to download only necessary blocks for vulnerability detection.
 
+If you want to scan a AMI of non-default setting region, you can set any region via `--region` option.
+
+```shell
+$ trivy vm --region ap-northeast-1 ami:ami-0123456789abcdefg
```

Comment:
VM scanning is not dedicated to AWS. We will probably add support for GCP and Azure.

```suggestion
$ trivy vm --aws-region ap-northeast-1 ami:ami-0123456789abcdefg
```